config/locales/en.yml

```diff
@@ -139,6 +139,11 @@ en:
     numeric:
       within_range: "%{field} must be between %{min} and %{max}"
       above_min: "%{field} must be at least %{min}"
+      whole_number: "%{field} must be a whole number"
+      nearest_ten: "%{field} must be given to the nearest ten"
+      nearest_penny: "%{field} must be given to the nearest penny"
```

Comment:
Are you sure this is always a monetary amount?

Reply:
I did check that currently all questions with step 0.01 are representing currency, but I agree this made me pause

Reply:
I think we need to keep in mind that numeric fields aren't limited to currency, and might not be in the future. Suggest changing `nearest_penny` to something that works for all numeric types.